src/normalize_attributes.cpp

```diff
@@ -117,11 +117,32 @@ auto tune_attribute(const std::vector<int64_t>& vec,
     return result;
 }
 
+auto tune_attribute(const value& val)
+{
+
+    std::vector<size_t> vec_attrs = val.to_vector<size_t>();
+    std::vector<size_t> result(vec_attrs.begin(), vec_attrs.end());
+    std::copy(vec_attrs.begin(), vec_attrs.end(), std::back_inserter(result));
+
+    return result;
+}
+
 bool normalize_attributes(operation& op, const std::vector<std::size_t>& lens)
 {
     bool tuned = false;
     auto attrs = op.attributes();
     auto val   = op.to_value();
+    if(attrs.contains("normalize_padding"))
+    {
+        if(val.at("padding").to_vector<size_t>().size() == 2 * (lens.size() - 2))
+            return true;
+        if(val.at("padding").to_vector<size_t>().size() != (lens.size() - 2))
+            MIGRAPHX_THROW("inconsistent padding size");
+        auto result    = tune_attribute(val.at("padding"));
```

Comment:
This should use the name passed to `normalize_padding`.

Reply:
You probably should do `auto padding = val.at(attrs.at("normalize_padding").to<std::string>())` and then replace `v.at("padding")` everywhere with `padding` variable.